packages/interactive-code-block/project.json

```diff
@@ -14,7 +14,6 @@
 				"cwd": "dist/packages",
 				"commands": [
 					"cp ../../packages/interactive-code-block/README.md ./interactive-code-block/",
-					"rm interactive-code-block/assets/php_5*",
```

Comment:
@adamziel this line was breaking the build. I assume we don't support PHP 5 anymore, so I removed it.

Let me know if I should move this to a new PR.

Reply:
Good call removing it, it's fine to do it in this PR